### DISCUSSION TITLE
strimzi operator kafka connect and rest api usage

### DISCUSSION THREAD

**Discussion by cstmgl** [strimzi/strimzi-kafka-operator#6502] [Q&A] [answered]
Hi all,
I'm again with one of my questions that might be basic but is not really for me.

I'm facing a bit of a problem when using Strimzi and KafkaConnector mainly because sometimes I want to do quick run-time verifications and Strimzi is too fast and destroys any modification I do on the REST API of KafkaConnect.

For example imagine a productive environment or close to it where you have limited access. I like to be able to quickly modify the admin/loggers to change for example some level to debug temporarily just to investigate some issue.
The point is that if I do this, Strimzi will act very fast and change them back to what they were in the CRD deplyoment.

Is there any way I can say to Strimzi to ignore changes to certain configurations? For example in the loggers?

The other option I can think about requires me to change the configMap for the log and scale the application down and up and I would rather not do that. (Actually since the configMap is also generated by Strimzi I'm not even sure that would work).

Thanks for any feedback and best regards,

**Comment by scholzj** [comment, marked as answer]
When you use the operator, the custom resources are the main source of truth. So no, you cannot go behind its back and change things manually. You should go, change the custom resource and let the operator apply the change for you.

**Reply by cstmgl** [reply]
I expected such. thanks for the confirmation.
on the one end I really get it, but specially for the logging I really would rather manage it in a different way.
Do you know if I don't provide that log section in the KafkaConnect if I could then manage it on the rest api itself.

basically remove this section from the KafkaConnect CRD

```
  logging:
    type: inline
    loggers:
      connect.root.logger.level: "INFO"
```

just have no logging at all. maybe one day I'll give it a try

**Reply by scholzj** [reply]
No, there is no way to manage the logging your self. When you leave the section out, it will just mean its using the default configuration. so it will be still reconciling it, just with different values.
